TestProjectHotline/Tests/HotlineTests/HotlineTest.feature

```diff
@@ -0,0 +1,82 @@
+﻿Feature: HotlineTest
+
+Background: Pre-Conditions
+Given User go to 'HotLineEnv'
+
+#Scenario: User login on hotline resource 
+#	When User login with 'TestUserHotLine@gmail.com' email and 'TestUser' password
+#	Then User sees 'TestUser12' nickname instead of 'login' button
+
+Scenario: User search product
```

Comment:
best practice:
Scenario name should be one word, using underscore '_' or PascalCase